tests/meta/input.ml

```diff
@@ -0,0 +1,30 @@
+open Test_lib
+open Report
+
+(*
```

Comment:
This record-based definition of meta.json must be removed since we finally chose to follow the let%meta approach.